utils/rl.json

```
@@ -22,6 +22,14 @@
                     "url": "/downloads/swan-lake-release-notes/swan-lake-2201.5.0",
                     "id": "swan-lake-2201.5.0"
                 },
+                {
+                    "dirName": "2201.4.2 (Swan Lake Update 4)",
```

```suggestion
                    "dirName": "2201.4.2 (Swan Lake Update 4)",
```
We need to add 2201.4.1 to the archived versions page also @sahanHe.

---

it's already there right? https://github.com/ballerina-platform/ballerina-dev-website/blob/ee658383eaeaed006e91640a4143de448a5a35ab/utils/archived-lm.json#L26